src/Faker/Provider/es_MX/Person.php

```diff
@@ -0,0 +1,337 @@
+<?php
+
+namespace Faker\Provider\es_MX;
+
+class Person extends \Faker\Provider\Person
+{
+    protected static $maleNameFormats = array(
+        '{{firstNameMale}} {{lastNameFather}} {{lastNameMother}}',
+    );
+
+    protected static $femaleNameFormats = array(
+        '{{firstNameFemale}} {{lastNameFather}} {{lastNameMother}}',
+    );
+
+    protected static $stateAbbr = array(
+        'AS', 'BC', 'BS', 'CC', 'CL', 'CM', 'CS', 'CH', 'DF', 'DG', 'GT', 'GR', 'HG', 'JC', 'MC', 'MN', 'MS', 'NT', 'NL', 'OC', 'PL', 'QT', 'QR', 'SP', 'SL', 'SR', 'TC', 'TS', 'TL', 'VZ', 'YN', 'ZS'
+    );
+
+    /**
+     * {@link} http://www.studentsoftheworld.info/penpals/stats.php3?Pays=MEX
```

Comment:
this page mentions a copyright. You need to use data from a MIT compatible datasource.